code-of-conduct.md

```diff
@@ -0,0 +1,147 @@
+- Abusive behavior is never tolerated.
+- Violations of this code may result in swift and permanent expulsion from the Quill community.
+- "Too long, didn't read" is not a valid excuse for not knowing what is in this document.
+
+### Table of Contents
+
+- [Scope](#scope)
+- [Friendly Harassment-Free Space](#friendly-harassment-free-space)
+- [Reporting Violations of this Code of Conduct](#reporting-violations-of-this-code-of-conduct)
+- [Copyright Violations](#copyright-violations)
+- [Consequences](#consequences)
+- [Addressing Grievances](#addressing-grievances)
+- [Contact Info](#contact-info)
+- [Changes](#changes)
+- [Credit and License](#credit-and-license)
+
+### Scope
+We expect all members of the Quill community to abide by this Code of Conduct at
+all times in all Quill community venues, online and in person, and in one-on-one
+communications pertaining to Quill affairs.
+
+This policy covers the usage of Quill public infrastructure, including the
+Quill.is HTTP gateways, as well as other Quill websites, Quill related events,
+and any other services offered by or on behalf of the Quill community. It also
+applies to behavior in the context of the Quill Open Source project communities,
+including but not limited to public GitHub repositories, Slack teams, IRC
+channels, social media, mailing lists, and public events.
+
+This Code of Conduct is in addition to, and does not in any way nullify or
+invalidate, any other terms or conditions related to use of Quill services.
+
+The definitions of various subjective terms such as "discriminatory", "hateful",
+or "confusing" will be decided at the sole discretion of the Quill abuse team.
+
+### Friendly Harassment-Free Space
+We are committed to providing a friendly, safe and welcoming environment for
+all, regardless of gender identity, sexual orientation, disability, ethnicity,
+religion, age, physical appearance, body size, race, or similar personal
+characteristics.
+
+We ask that you please respect that people have differences of opinion regarding
+technical choices, and that every design or implementation choice carries a
+trade-off and numerous costs. There is seldom a single right answer. A
+difference of technology preferences is not a license to be rude.
+
+Any spamming, trolling, flaming, baiting, or other attention-stealing behavior
+is not welcome, and will not be tolerated.
+
+Harassing other users is never tolerated, whether via public or private media.
+
+Avoid using offensive or harassing nicknames, or other identifiers that might
+detract from a friendly, safe, and welcoming environment for all.
+
+Harassment includes, but is not limited to: harmful or prejudicial verbal or
+written comments related to gender identity, sexual orientation, disability,
+ethnicity, religion, age, physical appearance, body size, race, or similar
+personal characteristics; inappropriate use of nudity, sexual images, and/or
+sexually explicit language in public spaces; threats of physical or non-
```

Comment:
We'll reserve the right for our moderation team to make decisions in this area, if and when necessary.